### ISSUE TITLE
Adding more Hyperparameters to LearningWithNoisyLabels()

### ISSUE THREAD

**Issue by alexriedel1** [cleanlab/cleanlab#58]
Hey there,
I'm facing the following situation:
Training a Model for k-Folds for n-Epochs. The Training over the Epochs is defined in my classifiers `fit()` function. With this being defined, `estimate_cv_predicted_probabilities()` willl also run for n-Epochs for each cv-fold.

Do you think it makes any sense to lower the number of Epochs for getting the `psx` from  `estimate_cv_predicted_probabilities()`, which would mean to change the number of epochs specifically for getting the noise labels, and then learn the full amount of epochs on the clean labels?

Thanks!

**Comment by cgnorthcutt** [comment]
I think that's a great idea - would you like to submit a pull request: https://github.com/cgnorthcutt/cleanlab/pulls

**Comment by jwmueller** [comment]
This use-case is a special instance of the functionality requested here:
https://github.com/cleanlab/cleanlab/issues/82

We'll track the request in that thread.

**Comment by jwmueller** [comment]
FYI this is now supported in v2.0.0 by `CleanLearning.fit()` via the args: 
`clf_kwargs` and `clf_final_kwargs`

https://docs.cleanlab.ai/v2.0.0/cleanlab/classification.html#cleanlab.classification.CleanLearning.fit
